ff/src/lib.rs

```diff
@@ -174,6 +179,7 @@ pub trait PrimeFieldRepr:
     }
 
     /// Writes this `PrimeFieldRepr` as a little endian integer.
+    #[cfg(feature = "std")]
     fn write_le<W: Write>(&self, mut writer: W) -> io::Result<()> {
```

Comment:
This API will be replaced with `no_std`-compatible `Field::to_bytes` (matching the `bls12_381` and `jubjub` APIs) when we remove `PrimeFieldRepr`.